eng/common/pipelines/templates/steps/generate-sdk.yml

```diff
@@ -0,0 +1,172 @@
+parameters:
+  - name: SpecRepoOwner
+    type: string
+  - name: SpecRepoName
+    type: string
+  - name: SdkRepoOwner
+    type: string
+  - name: SdkRepoName
+    type: string
+  - name: IntegrationSdkRepoOwner
```

Comment:
what is this for?

Reply:
I'm curious about the same. If this is about enabling forks, why can't we use the existing paramters for that?